### PR TITLE
Scientific Events Ontology (SEO)

### PR DESCRIPTION
**Pull request by saidfathalla**
Dear all,
This is the Scientific Events Ontology (SEO), a reference ontology for modelling data about scientific events such as conferences, symposioums and workshops

### CODE REVIEW

**Comment by dgarijo** [comment]
@saidfathalla 
w3id repo is meant to host the redirections, not the actual OWL files you are redirecting to. Please remove those files and point to them in your server or repository

**Comment by dgarijo** [comment]
Also, you seem to have copied pasted the htaccess from another repo, and you are creating w3ids such as w3id.org/SemSur/browserquest

**Comment by davidlehn** [comment]
Also remove the html.  Are those SemSur files supposed to be in this request?  If they are, the redirects are setup for `https://w3id.org/SemSur/semsur`, which I'm guessing is wrong?

Ideally rebase or squash this to remove the extra commits, but we can squash things if needed when merging.

**Comment by saidfathalla** [comment]
@dgarijo @davidlehn 
I hope everything is fine now.

**Comment by davidlehn** [comment]
- This is changing the `un/.htaccess` file.
-   Also just to check, since it wasn't implied by the title, you are adding the SemSur directory on purpose here?
- Did you test these files?
- These rules will result in https://w3id.org/seo/seo and https://w3id.org/SemSur/semsur.  Is that what you want?

**Comment by saidfathalla** [comment]
I do not know how this change un/.htaccess file? thats why i revert deleting semsur.
What I need is only redirect to seo by the URL: https://w3id.org/seo .

**Comment by saidfathalla** [comment]
@davidlehn @dgarijo I hope everything is fine now. Please pay attention to this request due to my time limitation. Thanks

**Comment by davidlehn** [comment]
You are still changing `un/.htaccess`.  Please revert that change.
https://github.com/perma-id/w3id.org/pull/1308/files

Also confirm you really want those double `seo/seo` type paths.  In a directory called "foo" most people want to use a `^$` pattern for the current dir vs a sub-path `^foo$`.

**Comment by saidfathalla** [comment]
@davidlehn @dgarijo , I have changed the.htacces files of semsur and seo to redirected from https://w3id.org/semsur and https://w3id.org/seo  respectively. I also reverted the access to un/.htaccess.
is it OK now?

File: SemSur/.htaccess [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+#Rewrite rules for semsur vocabulary 
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ http://sda.tech/SemSur/Documentation/Semsur.html  [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.applicationrdf+xml.
```
Comment (davidlehn) [inline SemSur/.htaccess]: That pattern looks wrong.  Perhaps this would work:
```suggestion
RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.*
```

**Comment by saidfathalla** [comment]
@davidlehn Thanks for the suggestion. Is it OK now?

**Comment by saidfathalla** [comment]
@davidlehn @dgarijo , could you please check?

**Pull request merged**